src/EnergyPlus/DaylightingManager.cc

```diff
@@ -5393,8 +5353,8 @@ void GeometryTransformForDaylighting(EnergyPlusData &state)
                     }
                 }
             } // refPtNum
-        }
-    }
+        }     // daylighting control loop
```

Comment:
I'm going to be somewhat pedantic here (for a change). I love these closing brace comments on large if/for bodies, but the standard comment here should be `// for (controlNum)`, i.e., `// for (<induction variable>)`. 😁